can/io/blf.py

```diff
@@ -359,6 +359,13 @@ def _parse_data(self, data):
 class BLFWriter(FileIOMessageWriter):
     """
     Logs CAN data to a Binary Logging File compatible with Vector's tools.
+
+    .. note::
+        The `buffer_size` will always stay under the `max_container_size`, which
+        is enforced at the end of the `_add_object` function. If the
+        `max_container_size > :attr:io.logger.RotatingLogger.max_bytes` then
+        the resulting file size will be less than described due to an
+        unaccounted compression of the buffer when writing to the file.
```

Comment:
This note references implentation details. I think this could be quite confusing for new users who are reading the docs. 
Instead of this, i would prefer a note in the file_size argument: The resulting file_size is dependent on the log format implementation and cannot be guaranteed. Something like that...

Reply:
I've updated the help statement for the file size based on your input. The one thing I don't like about it much is that it may create a sense of uncertainty. When `file_size` is the limiting factor, even with `BLFWriter` the resulting files created should be similar in size to one another. I think the statement may convey a wrong message; that the file size may vary wildly.